examples/mwaa/environment.yaml

```
@@ -0,0 +1,256 @@
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "logs:CreateLogStream",
+              "logs:CreateLogGroup",
+              "logs:PutLogEvents",
+              "logs:GetLogEvents",
+              "logs:GetLogRecord",
+              "logs:GetLogGroupFields",
+              "logs:GetQueryResults"
+            ],
+            "Resource": [
+              "arn:aws:logs:us-west-1:${aws_account_id}:log-group:airflow-example-*"
+            ]
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "logs:DescribeLogGroups"
+            ],
+            "Resource": "*"
+          },
+          {
+            "Effect": "Allow",
+            "Action": "s3:GetAccountPublicAccessBlock",
+            "Resource": "*"
+          },
+          {
+            "Effect": "Allow",
+            "Action": "cloudwatch:PutMetricData",
+            "Resource": "*"
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "sqs:ChangeMessageVisibility",
+              "sqs:DeleteMessage",
+              "sqs:GetQueueAttributes",
+              "sqs:GetQueueUrl",
+              "sqs:ReceiveMessage",
+              "sqs:SendMessage"
+            ],
+            "Resource": "arn:aws:sqs:us-west-1:*:airflow-celery-*"
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "kms:Decrypt",
+              "kms:DescribeKey",
+              "kms:GenerateDataKey*",
+              "kms:Encrypt"
+            ],
+            "NotResource": "arn:aws:kms:*:${aws_account_id}:key/*",
+            "Condition": {
+              "StringLike": {
+                "kms:ViaService": "sqs.us-west-1.amazonaws.com"
+              }
+            }
+          }
+        ]
+      }
+---
+apiVersion: s3.aws.upbound.io/v1beta2
+kind: Bucket
+metadata:
+  annotations:
+    meta.upbound.io/example-id: mwaa/v1beta1/environment
+  labels:
+    testing.upbound.io/example-name: example_bucket
+  name: example-bucket-mwaa
+spec:
+  forProvider:
+    region: us-west-1
+---
+apiVersion: s3.aws.upbound.io/v1beta2
+kind: BucketVersioning
+metadata:
+  annotations:
+    meta.upbound.io/example-id: mwaa/v1beta1/environment
+  labels:
+    testing.upbound.io/example-name: example_versioning
+  name: example-versioning
+spec:
+  forProvider:
+    bucketSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example_bucket
+    region: us-west-1
+    versioningConfiguration:
+      status: Enabled
+---
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: SecurityGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: mwaa/v1beta1/environment
+  labels:
+    testing.upbound.io/example-name: example_security_group
+  name: example-security-group
+spec:
+  forProvider:
+    region: us-west-1
+    description: Allow TLS inbound traffic
+    name: allow_tls
+    tags:
+      Name: allow_tls
+    vpcIdRef:
+      name: DemoVpc
```

We should pass the correct name.

```suggestion
      name: example-vpc
```